config/application.rb

```diff
@@ -179,9 +179,8 @@ class Application < Rails::Application
 # Must be here instead of in an initializer because plugin initializers run before app initializers
 # Used in plugins/airbrake + rollbar which do not support the 'foo.bar' syntax as rails does
 # https://github.com/airbrake/airbrake-ruby/issues/137
-Samson::Application.config.session_key = :"_samson_session_#{Rails.env}"
```

Comment:
keep this unless there is no `session` usage in any controller

Reply:
Yup, I checked we are not using any sessions, there may be session used in some gems.